UPGRADE-3.1.md

```diff
@@ -107,7 +107,7 @@ FrameworkBundle
    framework:
        cache:
            pools:
-               serializer:
+               cache.serializer:
```

Comment:
After re-reading https://github.com/symfony/symfony/pull/18667#issuecomment-215855491I think we can stop the discussion here as it was based on a misunderstand on my end.

👍 for how the PR is right now.